### ISSUE TITLE
Can I use system sound in local notifcation?

### ISSUE THREAD

**Issue by daheeahn** [zo0r/react-native-push-notification#1687]
Hi.

Currently, ios and android are using each custom sound by putting it in a folder directly. 
But what should I do when I want to use the sound built into the system?
If there is no way, please make it plz!

**Comment by rynvva** [comment]
@daheeahn do you have try soundName 'default' ?

<img width="309" alt="Screen Shot 2020-10-05 at 13 23 32" src="https://user-images.githubusercontent.com/39117076/95046270-1056e980-070e-11eb-8cb4-42e60517f0e8.png">

**Comment by daheeahn** [comment]
@rynvva Yes. But if I set it as default, only the default is imported. I want to use other alarm sounds in the system!
That means I don't put mp3 in the folder directly, but use the sound in the system.

**Comment by MoronovArt** [comment]
same issue
